src/MissionManager/SurveyMissionItem.cc

```diff
@@ -1062,7 +1062,7 @@ bool SurveyMissionItem::_appendMissionItemsWorker(QList<MissionItem*>& items, QO
         MissionItem* item = new MissionItem(seqNum++,
```

Comment:
Refly does not actually "refly" the survey. It just generates additional paths on the single survey.